src/main/java/org/tdl/vireo/model/repo/impl/SubmissionRepoImpl.java

```diff
@@ -1091,13 +1091,13 @@ public void setColumnOrderingForMonthYearDateFormat(Sort sort, List<String> sqlA
                 value.append("CAST(REPLACE(").append(table).append(".value, ' ', ' 1, ') AS DATE) AS");
             }
 
-            value.append(table).append("_date");
+            value.append(" " + table).append("_date");
 
             if (!sqlAliasBuilders.contains(value.toString())) {
                 sqlAliasBuilders.add(value.toString());
             }
 
-            sqlOrderBysBuilder.append(table).append("_date ").append(sort.name()).append(",");
+            sqlOrderBysBuilder.append(" " + table).append("_date ").append(sort.name()).append(",");
```

Comment:
As stated in above comment, attempt to avoid literal string concatenation and instead use StringBuilder to add the space for consistency reasons.
```suggestion
            sqlOrderBysBuilder.append(" ").append(table).append("_date ").append(sort.name()).append(",");
```